core/components/atoms/helpText/HelpText.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import { InlineMessage, Text } from '@/index';
-
-export interface HelpTextProps {
+import classNames from 'classnames';
+import { BaseProps } from '@/utils/types';
+export interface HelpTextProps extends BaseProps {
```

Comment:
`BaseProps` comes with two props: `className` & `data-test` but currently `data-test` does not seems to work in this case. Please check this once